### PR TITLE
dumpling: add csv line terminator option

### PR DESCRIPTION
**Pull request by GMHDBJD**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
add csv line terminator option

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?
https://github.com/pingcap/tidb/pull/46983

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dumpling-overview.md [hunk 1]
```diff
@@ -360,6 +360,7 @@ SET GLOBAL tidb_gc_life_time = '10m';
 | --csv-delimiter | csv 文件中字符类型变量的定界符 | '"' |
 | --csv-separator | csv 文件中各值的分隔符，如果数据中可能有逗号，建议源文件导出时分隔符使用非常见组合字符| ','|
 | --csv-null-value | csv 文件空值的表示 | "\\N" |
+| --csv-line-terminator | csv 文件中行尾的表示，由于 bash 中不同的 quote 会应用不同的转义规则，如需指定 LF 为换行符，可使用类似语法 --csv-line-terminator $'\n' | "\\r\\n" |
```
Comment (hfxsd) [inline dumpling-overview.md:363]:
```suggestion
| --csv-line-terminator | csv 文件中表示行尾的换行符，将数据导出为 CSV 文件时，可以通过该参数传入所需的换行符。该参数支持 "\r\n" 和 "\n"。由于 bash 中不同的 quote 会应用不同的转义规则，如需指定 LF 为换行符，可使用类似语法 `--csv-line-terminator $'\n'`。| "\\r\\n" |
```

File: dumpling-overview.md [hunk 2]
```diff
@@ -360,6 +360,7 @@ SET GLOBAL tidb_gc_life_time = '10m';
 | --csv-delimiter | csv 文件中字符类型变量的定界符 | '"' |
 | --csv-separator | csv 文件中各值的分隔符，如果数据中可能有逗号，建议源文件导出时分隔符使用非常见组合字符| ','|
 | --csv-null-value | csv 文件空值的表示 | "\\N" |
+| --csv-line-terminator | csv 文件中表示行尾的换行符，将数据导出为 CSV 文件时，可以通过该参数传入所需的换行符。该参数支持 "\r\n" 和 "\n"。由于 bash 中不同的 quote 会应用不同的转义规则，如需指定 LF 为换行符，可使用类似语法 `--csv-line-terminator $'\n'`。| "\r\n" |
```
Comment (Frank945946) [inline dumpling-overview.md:363]:
```suggestion
| --csv-line-terminator | csv 文件中表示行尾的换行符，将数据导出为 CSV 文件时，可以通过该参数传入所需的换行符。该参数支持 "\r\n" 和 "\n"，默认值为"\r\n" 和历史版本保持一致。由于 bash 中不同的 quote 会应用不同的转义规则，如需指定 LF 为换行符，可使用类似语法 `--csv-line-terminator $'\n'`。| "\r\n" |
```

**Review by hfxsd — APPROVED** [review]

**Review by Frank945946 — APPROVED** [review]

File: dumpling-overview.md [hunk 3]
```diff
@@ -360,6 +360,7 @@ SET GLOBAL tidb_gc_life_time = '10m';
 | --csv-delimiter | csv 文件中字符类型变量的定界符 | '"' |
 | --csv-separator | csv 文件中各值的分隔符，如果数据中可能有逗号，建议源文件导出时分隔符使用非常见组合字符| ','|
 | --csv-null-value | csv 文件空值的表示 | "\\N" |
+| --csv-line-terminator | csv 文件中表示行尾的换行符。将数据导出为 CSV 文件时，可以通过该参数传入所需的换行符。该参数支持 "\r\n" 和 "\n"，默认值为 "\r\n"，和历史版本保持一致。由于 bash 中不同的 quote 会应用不同的转义规则，如需指定 LF 为换行符，可使用类似语法 `--csv-line-terminator $'\n'`。| "\r\n" |
```
Comment (hfxsd) [inline dumpling-overview.md:363]:
```suggestion
| --csv-line-terminator | csv 文件中表示行尾的换行符。将数据导出为 CSV 文件时，可以通过该参数传入所需的换行符。该参数支持 "\r\n" 和 "\n"，默认值为 "\r\n"，和历史版本保持一致。由于 bash 中不同的 quote 会应用不同的转义规则，如需指定 LF 为换行符，可使用类似 `--csv-line-terminator $'\n'` 的语法。| "\r\n" |
```

File: dumpling-overview.md [hunk 4]
```diff
@@ -360,6 +360,7 @@ SET GLOBAL tidb_gc_life_time = '10m';
 | --csv-delimiter | csv 文件中字符类型变量的定界符 | '"' |
 | --csv-separator | csv 文件中各值的分隔符，如果数据中可能有逗号，建议源文件导出时分隔符使用非常见组合字符| ','|
 | --csv-null-value | csv 文件空值的表示 | "\\N" |
+| --csv-line-terminator | csv 文件中表示行尾的换行符。将数据导出为 CSV 文件时，可以通过该参数传入所需的换行符。该参数支持 "\r\n" 和 "\n"，默认值为 "\r\n"，和历史版本保持一致。由于 bash 中不同的 quote 会应用不同的转义规则，如需指定 LF 为换行符，可使用类似 `--csv-line-terminator $'\n'` 的语法。| "\r\n" |
```
Comment (qiancai) [inline dumpling-overview.md:363]:
```suggestion
| --csv-line-terminator | CSV 文件中表示行尾的换行符。将数据导出为 CSV 文件时，可以通过该选项传入所需的换行符。该选项支持 "\r\n" 和 "\n"，默认值为 "\r\n"，和历史版本保持一致。由于 bash 中不同的引号会应用不同的转义规则，如需指定 LF 为换行符，可使用类似 `--csv-line-terminator $'\n'` 的语法。| "\r\n" |
```

Comment (qiancai) [reply dumpling-overview.md:363]: L328，第一列是主要“选项”，所以参数可以修改为选项

**Review by qiancai — APPROVED** [review]
Rest LGTM

File: dumpling-overview.md [hunk 5]
```diff
@@ -360,6 +360,7 @@ SET GLOBAL tidb_gc_life_time = '10m';
 | --csv-delimiter | csv 文件中字符类型变量的定界符 | '"' |
 | --csv-separator | csv 文件中各值的分隔符，如果数据中可能有逗号，建议源文件导出时分隔符使用非常见组合字符| ','|
 | --csv-null-value | csv 文件空值的表示 | "\\N" |
```
Comment (hfxsd) [inline dumpling-overview.md:362]:
```suggestion
| --csv-delimiter | CSV文件中字符类型变量的定界符 | '"' |
| --csv-separator | CSV文件中各值的分隔符，如果数据中可能有逗号，建议源文件导出时分隔符使用非常见组合字符| ','|
| --csv-null-value | CSV 文件空值的表示 | "\\N" |
```

**Pull request merged**